ortools.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ortools": "v9.2-rte1.2"
```

Comment:
Shouldn't it be v9.2-rte1.3 ? There is a mismatch with .yml files
Is this file necessary ?

Reply:
I added file `ortools_tag` to the repository. I could have added a JSON file, but I need CMake to parse it. Parsing JSON is natively supported since Cmake 3.19, which is not easily available in Ubuntu 20.04.